.golangci.yml

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+run:
+  concurrency: 4
+  deadline: 3m
+  issues-exit-code: 1
+  tests: true
+  build-tags: [""]
+  skip-dirs:  [""]
+
+output:
+  format: colored-line-number
+  print-issued-lines: true
+  print-linter-name: true
+
+linters-settings:
+  errcheck:
+    check-type-assertions: true
+    check-blank: true
+  govet:
+    check-shadowing: true
+  golint:
+    min-confidence: 0.6
+  gofmt:
+    simplify: true
+  goimports:
+    local-prefixes: github.com/hyperledger/aries-framework-go-ext
+  gocyclo:
+    min-complexity: 10
+  maligned:
+    suggest-new: true
+  dupl:
+    threshold: 500
+  goconst:
+    min-len: 3
+    min-occurrences: 3
+  misspell:
+    # default locale is a neutral variety of English.
+    locale:
+    ignore-words: []
+  lll:
+    line-length: 120
+    tab-width: 1
+  unused:
+    check-exported: false
+  unparam:
+    check-exported: false
+  nakedret:
+    max-func-lines: 0
+  gocritic:
+    enabled-tags:
+      - diagnostic
+      - performance
+      - style
+      - opinionated
+    disabled-checks:
+      - whyNoLint # TODO enable.
+  funlen:
+    lines: 60
+    statements: 40
+  wsl:
+    strict-append: true
+    allow-assign-and-call: true
+    allow-multiline-assign: true
+    allow-case-traling-whitespace: true
+    allow-cuddle-declarations: false
+  godot:
+    check-all: false
+
+linters:
+  enable-all: true
+  disable:
+    - testpackage # TODO enable
+    - goerr113 # TODO enable
+    - nlreturn # TODO enable
+
+issues:
+  exclude-use-default: false
+  exclude-rules:
+    - path: _test\.go
+      linters:
+        - dupl
+        - funlen
+        - gomnd
+
+  exclude:
+    -  Line contains TODO/BUG/FIXME
+    # Add comments for package
+    -  at least one file in a package should have a package comment
```

Comment:
Can we remove this from exclude list ? Since we are starting from scratch, we should add package documentation. I believe, this was added to ignore list as we had lot of code in aries-framework-go and it was hard to fix all the lint issues.

Reply:
@troyronda @soluchok ^